docs/help/FAQ/Meltdown_and_Spectre.rst

```diff
@@ -0,0 +1,18 @@
+Meltdown and Spectre
```

Comment:
Should this go here https://simp.readthedocs.io/en/master/vulnerability_supplement/index.html